content/02.introduction.md

```diff
@@ -30,49 +31,62 @@ We plan to refine and expand this document until technologies to mitigate the pa
+The first coronaviruses that infect humans (hCoV) were identified in the 1960s [@doi:10.1136/bmj.1.5448.1467; @doi:10.3181/00379727-121-30734].
+Both of these viruses cause cold-like symptoms [@doi:10.1186/1743-422X-1-7; @pmid:20700397].
+No other coronaviruses infecting humans were identified until almost forty years later, when a new repiratory infection, which came to be known as Severe Acute Respiratory Syndrome (SARS), emerged [@doi:10.1038/nrmicro.2016.81].
+The first case was reported in November 2002 in the Guangdong Province of China, and over teh following month, the disease spread more widely wthin China and then into several countries across multiple continents [@doi:10.1093/ajcp/aqaa029; @doi:10.1038/nrmicro.2016.81]].
+SARS was both severe, with an estimated death rate of 9.5%, and highly contagious via droplet transmission, with a basic reproduction number of four (i.e., each person infected was estimated to infect four other people) [@doi:10.1093/ajcp/aqaa029].
+However, the identity of the virus behind the infection remained unknown until April of 2003, when the SARS-CoV virus was identified through a worldwide scientific effort spearheaded by the WHO [@doi:10.1038/nrmicro.2016.81].
+SARS-CoV belonged to a distinct lineage from the two other hCoV known at the time [@doi:10.1093/ajcp/aqaa029].
+By July 2003, the SARS outbreak was officially determined to be under control, with the success credited to successful infection management practices [@doi:10.1038/nrmicro.2016.81].
+A decade later, a second outbreak of severe respiratory illness associated with a coronavirus emerged, this time in the Arabian Peninsula.
+This disease, known as Middle East Respiratory Syndrome (MERS), was linked to another novel coronavirus, MERS-CoV.
+The fatality rate associated with MERS is much higher than that of SARS, at almost 35%, but the disease is much less easily transmitted, with a basic reproduction number of one [@doi:10.1093/ajcp/aqaa029].
+Although MERS is still circulating, its low reproduction number has allowed for its spread to be contained [@doi:10.1093/ajcp/aqaa029].
+Subsequently, two additional coronaviruses were identified that infect humans [@doi:10.1038/nm1024; @doi:10.1128/JVI.79.2.884-895.2005].
+In 2003, HCoV-NL63 [@doi:10.1038/nm1024] was identified in a 7-month-old infant and then identified in clinical specimens collected from seven additional patients, five of whom were infants (younger than 1 year) and the remainder of whom were adults.
+CoV-HKU1 was identified in samples collected from a 71-year-old pneumonia patient in 2004 and identified in samples collected from a second adult patient [@doi:10.1128/JVI.79.2.884-895.2005].
+The COVID-19 pandemic is thus associated with the seventh hCoV to be identified and the fifth since the turn of the millenium, though additional hCoVs in circulation may remain undetected.
```

Comment:
```suggestion
The COVID-19 pandemic is thus associated with the seventh hCoV to be identified and the fifth since the turn of the millennium, though additional, undetected hCoVs are likely to be in circulation.
```
Thank you for this suggestion! Does this rephrasing work? I definitely don't want this section to make it sound like the coronaviruses are suddenly a bigger problem, as I suspect it's just that we've figured out how to find them now.

Reply:
I agree, looks good to me